api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/HctDataConverter.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.ugandaemrreports.data.converter;
+
+import org.openmrs.Obs;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+
+/**
+ * Created by codehub on 7/11/17.
+ */
+public class HctDataConverter implements DataConverter {
+    @Override
+    public Object convert(Object obj) {
+
+        if (obj == null) {
+            return "";
+        }
+
+        Obs obs = ((Obs) obj);
+        if(obs.getValueCoded() != null && obs.getValueCoded().equals(Dictionary.getConcept("dc866728-30ab-102d-86b0-7a5022ba4115"))) {
```

Comment:
@ssmusoke the value of this **obs.getValueCoded()** is a concept, therefore I saw it faster to compare Concept Objects, but if it can save us anything by changing that check uuid, then I can change, but I see no difference